GOPKG_TOML_EXAMPLE.md

```diff
@@ -0,0 +1,60 @@
+# Gopkg.toml example
+
+## `[metadata]`
+ 1) Metadata defines metadata about the project that could be used by other independent systems. The metadata defined here will be ignored by dep.
+
+ 2) Metadata defines metadata about the dependency or override that could be used
+ by other independent systems. The metadata defined here will be ignored by dep.
+```toml
+[metadata]
+key1 = "value that convey data to other systems"
+system1-data = "value that is used by a system"
+system2-data = "value that is used by another system"
+```
+
+## `required`
+ Required lists a set of packages (not projects) that must be included in Gopkg.lock. This list is merged with the set of packages imported by the current project. Use it when your project needs a package it doesn't explicitly import - including "main" packages.
+ ```toml
+required = ["github.com/user/thing/cmd/thing"]
+```
+
+## `ignored`
+Ignored lists a set of packages (not projects) that are ignored when dep statically analyzes source code. Ignored packages can be in this project, or in a dependency.
+```toml
+ignored = ["github.com/user/project/badpkg"]
+```
+
+## `[[constraint]]`
+Constraints are rules for how directly imported projects may be incorporated into the depgraph. They are respected by dep whether coming from the Gopkg.toml of the current project or a dependency.
+```toml
+[[constraint]]
+  ## Required: the root import path of the project being constrained.
```

Comment:
my mistake, misread the comment. Was looking at the header of the markdown. Changing the toml comment now